### ISSUE TITLE
Add e-mail verification with 6-digit

### ISSUE THREAD

**Issue by resmo** [MasoniteFramework/masonite#464]
Sending links in HTML e-mails for verification is problematic due to spam and phishing protection, the e-mail often does not arrive.

Instead, I suggest the following mechanism: After registration, the user is redirected to a form that requires entering a 6-digit code. This numerical code is sent to the user by email.

**Comment by resmo** [comment]
it's not the email that's problematic, it's the link in it which could be seen as a phishing attack.

**Comment by girardinsamuel** [comment]
> I was going to close this but we might be able to do this through some classes you can build you own logic around

Yes it would be great. That's on those kind of features that we should pay attention to keep masonite 4 feature philosophy, simplicity and flexible !
